### DISCUSSION TITLE
v0.28.0

### DISCUSSION THREAD

**Discussion by borg323** [LeelaChessZero/lc0#1631] [General]
In this release:
* Multigather is now made the default (and also improved). Some search settings have changed meaning, so if you have modified values please discard them. Specifically, `max-collision-events`, `max-collision-visits` and `max-out-of-order-evals-factor` have changed default values, but other options also affect the search. Similarly, check that your GUI is not caching the old values.
* Updated several other default parameter values, including the MLH ones.
* Performance improvements for the cuda/cudnn backends. This includes the `multi_stream` cuda backend option that is off by default. You should test adding `multi_stream=true` to `backend-opts` (command line) or BackendOptions (UCI) if you have a recent GPU with a lot of VRAM. 
* Support for policy focus during training.
* Larger/stronger 15b default net for all packages except android, blas and dnnl that get a new 10b network.
* The distributed binaries come with the mimalloc memory allocator for better performance when a large tree has to be destroyed (e.g. after an unexpected move).
* The `legacy` time manager is again the default and will use more time for the first move after a long book line.
* The `--preload` command line flag will initialize the backend and load the network during startup. This may help in cases where the GUI is confused by long start times, but only if backend and network are not changed via UCI options.
* A 'fen' command was added as a UCI extension to print the current position.
* Experimental onednn backend for recent intel CPUs and GPUs.
* Added support for ONNX network files and runtime with the onnx backend.
* Several bug and stability fixes.

*Note:* Some small third-party nets seem to play really bad with the dx12 backend and certain GPU drivers, setting the `enable-gemm-metacommand=false` backend option is reported to work around this issue.

<hr /><em>This discussion was created from the release <a href='https://github.com/LeelaChessZero/lc0/releases/tag/v0.28.0'>v0.28.0</a>.</em>

**Comment by jerson-francisco** [comment]
i like leela zero

**Reply by jerson-francisco** [reply]
pls give me leela zero latest version

**Reply by borg323** [reply]
See https://github.com/LeelaChessZero/lc0/releases/tag/v0.31.2

**Comment by jerson-francisco** [comment]
#
